src/liballoc/collections/binary_heap.rs

```diff
@@ -165,6 +165,9 @@ use super::SpecExtend;
 /// trait, changes while it is in the heap. This is normally only possible
 /// through `Cell`, `RefCell`, global state, I/O, or unsafe code.
 ///
+/// Both `push` and `pop` operations can be performed in `O(log(n))` time, whereas `peek` can be
```

Comment:
This line should be break at `,` to conform with surrounding documentation's style.

Reply:
Thanks man! Fixed it!